main.go

```diff
@@ -2,23 +2,26 @@ package main
 
 import (
 	"github.com/murilogilfelpeto/ps-tag-onboarding-go/configuration"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/handler"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/repository"
 	"github.com/murilogilfelpeto/ps-tag-onboarding-go/router"
-)
-
-var (
-	logger *configuration.Logger
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/service"
 )
 
 func main() {
-	logger = configuration.GetLogger("main")
-	logger.Info("Starting application...")
-
-	err := configuration.Init()
+	appConfig, err := configuration.Init()
 	if err != nil {
-		logger.Errorf("Error initializing application: %v", err)
-		return
+		panic(err)
 	}
+	logger := appConfig.Logger
+
+	logger.Info("Starting application...")
+
+	userRepository := repository.NewRepository(appConfig.Database, "onboarding", "users")
+	userService := service.NewService(userRepository)
+	userHandler := handler.NewHandler(userService)
+	userRoute := router.NewRouter(userHandler)
```

Comment:
![image](https://github.com/murilogilfelpeto/ps-tag-onboarding-go/assets/91892156/a2f1ea30-6930-4ed8-9336-eb6da68f6314)

Reply:
Don't get it

Reply:
Don't get it

Reply:
Thumbs up! Good use of DI.